src/Components/Navbar/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState} from "react";
+import "./static/css/navbar.css";
+import logo from "./static/images/agric.png";
+import { FaSearch} from "react-icons/fa";
+import {IoIosArrowRoundForward} from 'react-icons/io'
+
+import {BrowserRouter as Router, Link } from "react-router-dom";
+import Search from './static/js/styled'
```

Comment:
create a file and name it styles.js in the Navbar folder, and put the styled component in it